OpenRA.Mods.Common/ServerTraits/PlayerPinger.cs

```diff
@@ -36,7 +36,7 @@ public void Tick(S server)
 				lastPing = Game.RunTime;
 
 				// Ignore client timeout in singleplayer games to make debugging easier
-				if (server.LobbyInfo.IsSinglePlayer && !server.Dedicated)
+				if (server.LobbyInfo.NonBotClients.Count() < 2 && !server.Dedicated)
```

Comment:
I kept it as suggested by pchote. It isn't possible in OpenRA IMO atm, but in theory there could be bot only game simulated.